src/app/opponents/components/opponents.component.ts

```diff
@@ -12,10 +12,16 @@ import { Observable } from 'rxjs';
   styleUrls: ['./opponents.component.scss']
 })
 export class OpponentsComponent {
+  @Input()
+  hasCurrentPlayerChosen = false;
```

Comment:
I don't think this needs to be an `Input` but can be gotten in here, leaving this component without that dependency. But let's merge for now, we can refactor later. Time to close all of these :)